client-src/elements/chromedash-roadmap.ts

```diff
@@ -2,6 +2,15 @@ import {LitElement, html, css} from 'lit';
 import {showToastMessage} from './utils.js';
 import '@polymer/iron-icon';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {customElement, property, state} from 'lit/decorators.js';
+import {Feature} from '../js-src/cs-client.js';
+
+interface MilestoneInfo {
+  version: number;
+  features: {
+    [key: string]: Feature[];
+  };
```

Comment:
I think you've misread the type of this object, which is merged between https://chromiumdash.appspot.com/fetch_milestone_schedule?mstone=120 (fetched from https://github.com/GoogleChrome/chromium-dashboard/blob/7f37bd7c9df24d3861cb694abe85d6bc8c6cb2e5/api/channels_api.py#L96-L103) and the `newMilestonesInfo` variable in this file.

It looks like it should be
```suggestion
  [milestone: number]: {
    // TODO: Add the relevant fields from https://chromiumdash.appspot.com/fetch_milestone_schedule?mstone=120
    version: number;
    features: Feature[];
  };
```

If you give `newMilestonesInfo` a more precise type than `any`, it'll help you keep track of this. And to do that, it might be good to convert `cs-client.js` to Typescript sooner rather than later.